dev/integration_tests/gradle_deprecated_settings/pubspec.yaml

```diff
@@ -16,7 +16,7 @@ dependencies:
   characters: 1.3.0 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
   collection: 1.17.1 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
   cross_file: 0.3.3+4 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
-  flutter_plugin_android_lifecycle: 2.0.8 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
+  flutter_plugin_android_lifecycle: 2.0.9 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
```

Comment:
@reidbaker could you take a look at this? This change appears to be causing the following failure (although I can't explain why it seems to have passed on Linux...)

```
[        ] Cause 1: org.gradle.internal.resolve.ModuleVersionResolveException: Could not resolve androidx.annotation:annotation:1.5.0.
[        ] Required by:
[        ]     project :flutter_plugin_android_lifecycle
```

https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8787684536026999009/+/u/run_test.dart_for_tool_integration_tests_shard_and_subshard_4_4/test_stdout

Here is the changelog for this plugin: https://github.com/flutter/packages/commits/main/packages/flutter_plugin_android_lifecycle

Reply:
actually, I think it's this: https://github.com/flutter/packages/pull/3271/files#diff-a7a667ae2f4a6b9d0a2fc91cc1eb18310cf7bc98cecbcd9b27fe444028d87887R37